### DISCUSSION TITLE
Improve Tooltips In Posts & Comments

### DISCUSSION THREAD

**Discussion by AyushmanOfficial** [dailydotdev/daily#1416] [Feature Requests]
Instead of displaying the string "see who upvoted" when a comment's tooltip is hovered over, why don't we display two or more names, similar to what Facebook does?

In comparison

Daily.dev:
![image](https://github.com/user-attachments/assets/eac8a64e-53cf-40ff-b34c-50911c4284f9)

Facebook:
![image](https://github.com/user-attachments/assets/87e2e61e-bc53-4817-bf8c-61d63ae5367a)

Do this for posts as well! when hovering over an upvoted post it does nothing.
![image](https://github.com/user-attachments/assets/beb6e06a-2b0b-443c-add8-f16277abe43d)

**Comment by idoshamun** [comment]
Hi @AyushmanOfficial,

Thanks for the suggestion :)

**Reply by AyushmanOfficial** [reply]
Thanks for the quick response :heart: :rocket: !
Is there any way to change the shortcut for the search bar aside Ctrl + K?

**Reply by idoshamun** [reply]
Not at the moment
